apps/server/test/lexical/server/code_intelligence/completion_test.exs

```diff
@@ -152,5 +158,38 @@ defmodule Lexical.Server.CodeIntelligence.CompletionTest do
       assert {:ok, _} = fetch_completion(completions, label: "Foo-protocol")
       assert {:ok, _} = fetch_completion(completions, label: "Foo-struct")
     end
+
+    test "modules are sorted before functions", %{project: project} do
```

Comment:
You mentioned in the document that local boost and global boost have different usage. Can we have more unit tests to demonstrate use cases?

Reply:
Great idea